packages/toaster/tests/Toast.test.tsx

```diff
@@ -1,92 +1,48 @@
 import * as React from "react";
-import { shallow, mount } from "enzyme";
+import { render, screen } from "@testing-library/react";
 import { createSerializer } from "@emotion/jest";
-import toJson from "enzyme-to-json";
-import { Toaster, Toast } from "../";
+import { Toast } from "../";
+import userEvent from "@testing-library/user-event";
 
 expect.addSnapshotSerializer(createSerializer());
 
 describe("Toast", () => {
   it("renders default", () => {
-    const component = shallow(
-      <Toaster>
-        <Toast title="I Am Toast" key={0} id={0} />
-      </Toaster>
-    );
-    expect(toJson(component)).toMatchSnapshot();
+    const { asFragment } = render(<Toast title="I Am Toast" key={0} id={0} />);
+    expect(asFragment()).toMatchSnapshot();
   });
 
   it("renders with description", () => {
-    const component = shallow(
-      <Toaster>
-        <Toast title="I Am Toast" description="Description" key={0} id={0} />
-      </Toaster>
+    const { asFragment } = render(
+      <Toast title="I Am Toast" description="Description" key={0} id={0} />
     );
-    expect(toJson(component)).toMatchSnapshot();
+    expect(asFragment()).toMatchSnapshot();
   });
 
   it("renders with actions", () => {
-    const component = shallow(
-      <Toaster>
-        <Toast
-          primaryAction={<button>primaryAction</button>}
-          secondaryAction={<button>secondaryAction</button>}
-          title="I Am Toast"
-          key={0}
-          id={0}
-        />
-      </Toaster>
+    const { asFragment } = render(
+      <Toast
+        primaryAction={<button>primaryAction</button>}
+        secondaryAction={<button>secondaryAction</button>}
+        title="I Am Toast"
+        key={0}
+        id={0}
+      />
     );
-    expect(toJson(component)).toMatchSnapshot();
+    expect(asFragment()).toMatchSnapshot();
   });
 
-  it("should call the function in the action when clicked", () => {
```

Comment:
I removed this test entirely because it was testing the functionality of the component passed as the `primaryAction`. It really has nothing to do with what is happening in Toast